data-raw/process_data.R

```diff
@@ -28,7 +28,7 @@ all <- do.call(rbind, all)
 # transform
 attr_cols <- names(all) %in% paste0("Source",0:3)
 all[,attr_cols] <- exp(all[,attr_cols])
-all[,attr_cols] <- all[,attr_cols] / rowSums(all[,attr_cols])
+all[,attr_cols] <- all[,attr_cols] / colSums(all[,attr_cols])
```

Comment:
Hmm, not sure if this is correct, are we sure that dividing by the column sum is what we want to do?

Need to look into this in a bit more detail...